DBSCE/Overview.md

```diff
@@ -0,0 +1,219 @@
+    - [Attestation Service:](#attestation-service)
+    - [Device Registration (Pre-Session)](#device-registration-pre-session)
+    - [DBSC(E) use cases](#dbsce-use-cases)
+      - [IDP Calls Public Local Key Helper](#idp-calls-public-local-key-helper)
+      - [IDP is RP and Calls Public Local Key Helper](#idp-is-rp-and-calls-public-local-key-helper)
+      - [IDP Calls Private Local Key Helper](#idp-calls-private-local-key-helper)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Device Bound Session Credentials for Enterprise - explainer
+
+This is the repository for Device Bound Session Credentials for Enterprise. You're welcome to
+[contribute](CONTRIBUTING.md)!
+
+## Authors
+
+- [Sameera Gajjarapu](sameera.gajjarapu@microsoft.com), Microsoft
+- [Alexander Tokarev](alextok@microsoft.com), Microsoft
+
+## Contributors
+
+- [Olga Dalton](), Microsoft
+- [Kristian Monsen](), Google
+- [Phil Leblanc](), Google
+- [Sebastian](), Google
+- [Arnar Birgisson](), Google
+- [Pamela Dingle](), Microsoft
+- [Paul Garner](), Microsoft
+- [Erik Anderson](), Microsoft
+- [Will Bartlett](), Microsoft
+- [Kai Song](), Microsoft
+- [Amit Gusain](), Microsoft
+
+## Participate (TBD links)
+
+- [Issue tracker]()
+- [Discussion forum]
+
+## Overview
+
+Device Bound Session Credentials for Enterprise - DBSC(E), is an enhancement to the existing [DBSC](https://github.com/wicg/dbsc) proposal. It refines the key generation mechanism resulting in additional security for enterprise use cases. It aims to provide a mechanism for enterprise and advanced browser customers to be able to deploy true device binding for any browser session, hence protecting against session hijacking and credential theft.
+
+## Why DBSC(E)?
+
+While the original DBSC proposal is focused on providing a mechanism for browsers to bind session credentials to a device, it still remains vulnerable to malware that can run on a device during the initial web application signin/login. If a malware happens to be already running in the device, it can force the user to login, and provide its own credentials (asymmetric key pair) to the web application, there by gaining the ability to steal the session. Any upcoming sessions after this, even with DBSC, will not be reliable.
+
+DBSC(E) aims to mitigate this risk by introducing the concept of credential generation (asymmetric device-bound key) during the device registration and binds all the future sessions to the device. It guaratees any given session to be bound to the device, if the device registration is performed when there is no malware on the device (a state referred to as "clean room"). With DBSC(E), the malware will not be able to compromise a device even during signin/login. Device registration is also expected to be a once-in-a-lifetime operation, and hence the user will not be required to perform this operation again, reducing opportunities for malware to extract session credentials the device.
```

Comment:
Maybe state an example of a clean room would be useful.  For instance, after a fresh install of an operating system, the user is asked to sign in with an existing account or create a new one to sign in with.  Once the user signs in, but before any other software is installed on the device, the device registration occurs.

Reply:
You call this out in a bit more detail on line 97 below.  This is a good example of a clean room.  "When the browser is installed"  (line 98) isn't necessarily a clear room or a device that is free from malware.

Reply:
Added link to `device registration client` section.